code/modules/cm_marines/smartgun_mount.dm

```diff
@@ -84,7 +86,11 @@
 
 /obj/item/device/m56d_gun/attack_self(mob/user)
 	..()
-
+	if(!can_be_near_defense)
+		for(var/obj/structure/machinery/machine in urange(defense_check_range, loc))
+			if(istype(machine, var/obj/structure/machinery/m56d_hmg) || istype(machine, var/obj/structure/machinery/m56d_post)
```

Comment:
Missing `)` and check your syntax of `istype()`

https://www.byond.com/docs/ref/#/proc/istype